build/ig.ini

```diff
@@ -1,7 +1,7 @@
 [IG]
 ig = input/ImplementationGuide-fhir.us.mcode.json
-template = hl7.fhir.template#0.0.2
-template = hl7.fhir.template#current
+template = hl7.fhir.template#0.0.5
+template = hl7.base.template#current
```

Comment:
Is this a workaround? The inclusion of the base template should happen automatically.